lambdatimes/src/components/Content/Cards.js

```diff
@@ -1,15 +1,29 @@
-import React, { Component } from 'react';
-import Card from './Card';
+import React, { Component } from "react";
+import Card from "./Card";
+import PropTypes from "prop-types";
 
 const Cards = props => {
+  console.log(props);
   return (
     <div className="cards-container">
-      {/* Using the cards prop, map over the list creating a 
+      {props.cards.map(card => (
+        <Card card={card} />
+      ))}
+      {/* Using the cards prop, map over the list creating a
           new Card component for each passing the card as the only prop*/}
     </div>
-  )
-}
+  );
+};
 
 // Make sure you include prop types for all of your incoming props
 
-export default Cards;
\ No newline at end of file
+Cards.propTypes = {
+  cards: PropTypes.shape({
+    author: PropTypes.string,
+    headline: PropTypes.string,
```

Comment:
This proptype is actually broken.. Cards is an array isn't it? So here you are saying it is an object, this is partially true, since cards is an array of objects with this shape.